src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -293,6 +250,7 @@ public void testInit() {
   @Override
   public void robotPeriodic() {
     Scheduler.getInstance().run();
+    // TODO: Move SmartDashboard puts to separate class
```

Comment:
We could also remove the smartdashboard put here entirely as it was just for debugging.

Reply:
Maybe. We do have the `@Telemetry` annotation but it would be cool to have that annotation on, say, a method (`getPosition()` etc.). Maybe create an issue in ROOSTER.